lib/spack/spack/test/cmd/env.py

```diff
@@ -525,6 +525,25 @@ def test_env_with_config():
                for x in e._get_environment_specs())
 
 
+def test_with_config_bad_include(capfd):
+    test_config = """\
+spack:
+  include:
+  - /no/such/directory
+  - no/such/file.yaml
+"""
+    _env_create('test', StringIO(test_config))
+
+    e = ev.read('test')
+    with e:
+        e.concretize()
+
+    _, err = capfd.readouterr()
+    assert 'Ignoring' in err
+    assert 'include /no/such/directory' in err
+    assert 'no/such/file.yaml' in err
```

Comment:
The test needs to be updated now that a missing include is a failure.

Reply:
That's what I get for making the fix "on the side" while trying to get something else done.

Reply:
Fixed and modified the PR title and description accordingly.

Reply:
Hmm.  It looks like this change is having a ripple affect on other unit tests.